backend/database/migrations/2022_04_06_025951_create_user_relationship_models_table.php

```diff
@@ -15,8 +15,8 @@ public function up()
     {
         Schema::create('user_relationships', function (Blueprint $table) {
             $table->bigIncrements('user_relationship_id');
-            $table->integer('following_id');
-            $table->integer('followed_id');
+            $table->integer('following_id')->nullable();
+            $table->integer('followed_id')->nullable();
```

Comment:
I think its better if this is not nullable?
In which scenario will this be applicable if ever

Reply:
I will edit this one, I was just overthinking a scenario and looks like I don't need it to be nullable